packages/getting-started/src/browser/getting-started-frontend-module.ts

```diff
@@ -29,4 +29,5 @@ export default new ContainerModule((bind: interfaces.Bind) => {
         id: GettingStartedWidget.ID,
         createWidget: () => context.container.get<GettingStartedWidget>(GettingStartedWidget),
     })).inSingletonScope();
+    bind(PreferenceContribution).toConstantValue({ schema: welcomePreferenceSchema });
```

Comment:
Let's implement the binding function similarly to other `*-preferences.ts` files, and add it to `getting-started-preferences.ts`.